BlockEntity/BEPitKiln.cs

```diff
@@ -499,6 +499,8 @@ public override void GetBlockInfo(IPlayer forPlayer, StringBuilder dsc)
 
             if (Lit) dsc.AppendLine(Lang.Get("Lit"));
             else dsc.AppendLine(Lang.Get("Unlit"));
+
+            base.GetBlockInfo(forPlayer, dsc);

```

Comment:
same here, this would display additional information

Reply:
BlockEntityBehavior can't override GetBlockInfo for pit kilns due to this